RadixWallet/Features/SplashFeature/Splash.swift

```diff
@@ -152,17 +159,35 @@ public struct Splash: Sendable, FeatureReducer {
 			}
 
 			return delegateCompleted(context: state.context)
+
+		case .showAppLockMessage:
+			state.destination = .errorAlert(.init(
+				title: .init("Warning"),
+				message: .init("To protect your app, use the App Lock system feature"),
```

Comment:
Is the copy final? I think "to protect your app"is too generic a message, especially with the title "Warning", it makes it sound like users are "unsafe" without it.

Probably not your copy! But I suggest you bring this up with Matt / Charlie. 

I would prefer more precise wording alluding to the missed out feature and the purpose of the lock, something along the lines of:
Title: "Recommendation"
Message: "For maximum privacy you might wanna consider enabling Lock Screen"

Reply:
Yes, already asked Matt